be/src/olap/rowset/segment_group.cpp

```diff
@@ -268,11 +265,14 @@ OLAPStatus SegmentGroup::add_zone_maps_for_linked_schema_change(
         WrapperField* second = WrapperField::create(column);
         DCHECK(second != NULL) << "failed to allocate memory for field: " << i;
 
-        if (schema_mapping[i].ref_column == -1) {
+        // add column also can add zone map index
+        if (schema_mapping[i].ref_column == -1 || i >= zone_map_fields.size()) {
             // ref_column == -1 means this is a new column.
             // for new column, use default value to fill into column_statistics
-            first->copy(schema_mapping[i].default_value);
-            second->copy(schema_mapping[i].default_value);
+            if (schema_mapping[i].default_value != nullptr) {
```

Comment:
if you set it to default value， in duplicated table update from 0.11 to 0.12 may cause value clumn filtered unexpected